sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 2.0.0-beta.2 (Unreleased)
+## 2.0.0-beta.2 (2021-04-06)
 
 - Added `clientId` to the `AuthenticationRecord` type.
```

Comment:
Can we add some details as to how this clientId would be helpful to the user? Same for the entry "Added properties `scopes` and `getTokenOptions` to the `AuthenticationRequired` error".

Reply:
The case for `clientId` is a bit complicated since it's an internally useless parameter that was added "just in case"  across languages. What do you think of?:

> - Added `clientId` to the `AuthenticationRecord` type. This will allow applications that de-serialize an `AuthenticationRecord` to recover the `clientId` used to do the original authentication.

On scopes and getTokenOptions on Authentication record, what do you think of?:

> - Added properties `scopes` and `getTokenOptions` to the `AuthenticationRequired` error. On the credentials that allow `disableAutomaticAuthentication`, if that constructor option is enabled, `getToken` won't automatically trigger manual authentication. If `getToken` fails to silently authenticate, it will throw an `AuthenticationRequired` error that will let developers know that a manual authentication must now be triggered with the `authenticate()` method. To make it easier to re-authenticate at any point, the error `AuthenticationRequired` now contains `scopes` and `getTokenOptions`, which can be passed as parameters to the `authenticate()` method.

Reply:
> Added clientId to the AuthenticationRecord type. This will allow applications that de-serialize an AuthenticationRecord to recover the clientId used to do the original authentication.

Client Id is something that the user passes to the credential in the first place. So, they have access to it already. Why would they need to access it from the authentication record? And if the idea is to re-do the original authentication, wont they need other details like tenantId or secret or something else based on the credential being used?

For scopes & getTokenOptions, how about this:

> Added properties `scopes` and `getTokenOptions` to the AuthenticationRequired error. These properties hold the values used by the `getToken()` method on your credential to fetch the access token. You should pass these to the `authenticate()` method on your credential if you wanted to do manual authentication after catching the `AuthenticationRequired` error.

Reply:
`AuthenticationRecord` already had a `tenantId`, this is to add the `clientId` alongside it. I'll mention that `tenantId` was already there. I'll update the PR in a bit.